### PR TITLE
Pull Request Attempt 2

### PR DESCRIPTION
**Pull request by taylormleigh**
The first pull request didn't go through; let me try again. All commits in this branch are front-end changes.
+Skeletons for: NavBar, FrontPage, UserProfile, LoginPage
+Custom Hooks in App.jsx from CustomHooks.js --> different callback functions being called inside the useForm function will apply form input to different server routes

### CODE REVIEW

**Comment by taylormleigh** [comment]
@TomGower it looks like there's a conflict with the schema file? I used the updated file from the upstream master so I'm not sure what the problem is but I'll wait for you to take a look at it :)

**Comment by TomGower** [comment]
Yeah, I fiddled with it because I was having trouble with the enum fields
and fake data, and some foreign key stuff. It shouldn't be a big deal
either way.
